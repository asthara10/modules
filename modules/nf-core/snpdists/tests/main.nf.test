nextflow_process {

    name "Test Process SNPDISTS"
    script "../main.nf"
    process "SNPDISTS"

    tag "modules"
    tag "modules_nfcore"
    tag "snpdists"

    test("test_snpdists") {


        when {
            process {
                """
                
                input[0] = [ [ id:'test', single_end:false ], // meta map
              file(params.test_data['sarscov2']['genome']['informative_sites_fas'], checkIfExists: true) ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
