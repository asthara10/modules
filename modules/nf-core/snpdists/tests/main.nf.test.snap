{
    "versions": {
        "content": [
            [
                "versions.yml:md5,62490c6d1f42c5d2eebeea9fcf78f619"
            ]
        ],
        "timestamp": "2023-11-24T12:56:57.055859"
    }
}